cpp/ps0003/p0056.cpp

```diff
@@ -4,17 +4,17 @@ using namespace std;
 
 int main()
 {
-   int number ;
-   cout << "enter number \n" ;
-   cin >> number ;
+   // prompting user for number 2/2
+   int number;
+   cout << "enter number \n";
+   cin >> number;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number